ui/selectors/transactions.js

```diff
@@ -103,8 +103,18 @@ export const smartTransactionsListSelector = (state) => {
     getCurrentChainId(state)
   ]
     ?.filter((stx) => {
-      const { txParams } = stx;
-      return txParams?.from === selectedAddress && !stx.confirmed;
+      const isCancelledSmartTransaction = stx.status?.startsWith('cancelled');
```

Comment:
Is the `status` a dynamic `string` or some kind of enum?

In which case should we be explicit and compare against a specific of status values?

Reply:
There are currently 7 different `cancel` statuses in the STX controller, which all start with the `cancel` string. I used an existing code that is a few lines below this one, but sure, we can check against a specific list of `cancel` statuses.

Reply:
I could create a new function in the STX controller, e.g. `isCancelledSmartTransaction` and use it in a future PR.